builtin/providers/aws/resource_aws_s3_bucket.go

```diff
@@ -927,6 +1274,36 @@ func normalizeRegion(region string) string {
 	return region
 }
 
+func expirationHash(v interface{}) int {
```

Comment:
Will one of these params always be part of the hash? If not, what will happen with the hash?